src/darwin/Framework/CHIP/templates/MTRClusters-src.zapt

```diff
@@ -75,7 +76,7 @@ MTR{{cluster}}Cluster{{command}}Params
 {{/inline}}
 {{#*inline "clusterId"}}
 {{#if (wasRemoved cluster command=command)}}
-{{asMEI ../manufacturerCode ../code}}
+{{asMEI ../../manufacturerCode ../../code}}
```

Comment:
Why was the change needed?  Where is the extra scope coming from compared to before?

Reply:
It seems to be coming from the arguments to `commandImpl` when used within `zcl_commands`
In `commandImpl` I see the following:
```this : MfgSpecificPing 0x1002 0x0; -> command
parent : MfgSpecificPing 0x1002 0x0; -> command
parent.parent : Basic Information 0x0 0x28; ->cluster```

Reply:
Yes, but why is it different from the other method of iterating over commands?

Reply:
Filed https://github.com/handlebars-lang/handlebars.js/issues/1931 based on our discussion in https://csamembers.slack.com/archives/C01B01GH64E/p1680186533930189